addons/mail/wizard/mail_compose_message.py

```diff
@@ -298,8 +299,8 @@ def get_mail_values(self, res_ids):
                 'parent_id': self.parent_id and self.parent_id.id,
                 'partner_ids': [partner.id for partner in self.partner_ids],
                 'attachment_ids': [attach.id for attach in self.attachment_ids],
-                'author_id': self.author_id.id,
```

Comment:
So here, you don't use the author_id information from the wizard itself right ? 

Is it really intended ? The fields should still be displayed (if it currently is).